website/docs/agent-installation.md

```diff
@@ -4,12 +4,12 @@ title: Litmus Chaos Agent Install
 sidebar_label: Chaos Agent
 ---
 
-In Litmus the Agents can be classified as two types
+In Litmus the Agents can be classified into two, namely;
 
 - Self Agent
 - External Agent
 
-As part of Litmus installation by default, a self cluster would be registered as Agent in the Portal. From Portal you induce chaos into self cluster and observe the results from the Portal.
+As a part of the Litmus installation, by default, a self cluster would be registered as Agent in the Portal. From the Portal you induce chaos into self cluster and observe the results from the Portal.
```

Comment:
As a part of the Litmus installation, by default, a self cluster would be registered as an Agent in the Portal. From the Portal you may induce chaos into self cluster and observe the results from the Portal.